openbas-api/src/test/java/io/openbas/rest/AtomicTestingApiTest.java

```diff
@@ -38,39 +41,21 @@ public class AtomicTestingApiTest extends IntegrationTest {
 
   @BeforeAll
   void beforeAll() {
-    Inject injectToCreate1 = new Inject();
-    injectToCreate1.setTitle("Inject without payload");
-    injectToCreate1.setCreatedAt(Instant.now());
-    injectToCreate1.setUpdatedAt(Instant.now());
-    injectToCreate1.setDependsDuration(0L);
-    injectToCreate1.setEnabled(true);
-    injectToCreate1.setInjectorContract(
-        injectorContractRepository.findById(EMAIL_DEFAULT).orElseThrow());
-    INJECT_WITHOUT_PAYLOAD = injectRepository.save(injectToCreate1);
-
-    Inject injectToCreate2 = new Inject();
-    injectToCreate2.setTitle("Inject with payload");
-    injectToCreate2.setCreatedAt(Instant.now());
-    injectToCreate2.setUpdatedAt(Instant.now());
-    injectToCreate2.setDependsDuration(0L);
-    injectToCreate2.setEnabled(true);
-    injectToCreate2.setInjectorContract(
-        injectorContractRepository.findById(EMAIL_DEFAULT).orElseThrow());
-    INJECT_WITH_PAYLOAD = injectRepository.save(injectToCreate2);
-    InjectStatus injectStatus = new InjectStatus();
-    injectStatus.setInject(injectToCreate2);
-    injectStatus.setTrackingSentDate(Instant.now());
-    injectStatus.setName(ExecutionStatus.SUCCESS);
-    injectStatus.setCommandsLines(
-        new InjectStatusCommandLine(List.of("cmd"), List.of("clean cmd"), "id1234567"));
+    INJECTOR_CONTRACT = injectorContractRepository.findById(EMAIL_DEFAULT).orElseThrow();
+    Inject injectWithoutPayload = InjectFixture.getInjectForEmailContract(INJECTOR_CONTRACT);
+    INJECT_WITHOUT_PAYLOAD = injectRepository.save(injectWithoutPayload);
+
+    Inject injectWithPayload = InjectFixture.getInjectForEmailContract(INJECTOR_CONTRACT);
+    INJECT_WITH_PAYLOAD = injectRepository.save(injectWithPayload);
```

Comment:
I improved the naming to reflect if the status is present or not.
This naming was existing code, not part of the scope of my PR ^^"
Initially, I only extracted several fixtures and remove the ordering in the tests since not necessary